cli/dinghy/http_proxy.rb

```diff
@@ -41,8 +41,17 @@ def up(expose_proxy: true)
       "--name", CONTAINER_NAME, IMAGE_NAME)
```

Comment:
Oh this is still incorrect, in my previous comment I meant that you should revert all changes to this file.